lib/landrush/command.rb

```diff
@@ -63,7 +67,7 @@ def help; <<-EOS.gsub(/^      /, '')
         set { <host> <ip> | <alias> <host> }
           adds the given host-to-ip or alias-to-hostname mapping.
           existing host ip addresses will be overwritten
-        rm, del { <host> | <alias> }
+        rm, del { <host> | <alias> | --all }
```

Comment:
I was actually going to do that in my initial commit, but after thinking about it more, that would make `landrush rm --all <host>` valid per the `help` page, but `landrush rm <host> --all` would be invalid, which is kind of confusing.

Not to mention, the host being there is kind of irrelevant, since the `--all` flag is what we actually care about and will be checking for. What you suggested makes it look like a host/alias is required with an optional `--all` flag, which isn't the case.

What we're looking for is actually either or, not both. Right?